app/models/adoption_application.rb

```diff
@@ -0,0 +1,25 @@
+class AdoptionApplication < ApplicationRecord
+   has_many :adoption_application_pets
+   has_many :pets, through: :adoption_application_pets
+
+   validates :name, presence: true 
+   validates :street_address, presence: true
+   validates :city, presence: true 
+   validates :state, presence: true 
+   validates :zip_code, presence: true, numericality: true
+   validates :description, presence: true
+
+   def add_pet_to_app(pet_id)
+      pet = Pet.find(pet_id)
+      self.pets << pet
+   end
+
+   # admin 
+   def change_app_status(new_status)
+      self.update(status: new_status)
+   end
+
+   def add_ownership_description_to_app(description)
+      self.update(ownership_description: description)
+   end
```

Comment:
Model looks good! These model methods are nice and readable. It's okay, however, to use `update` and `find` in controllers, though, just for the future! There's nothing wrong with further encapsulating this logic in a method like you have done here as long as you test it in the model spec.